### PR TITLE
refactor: update `offset` handling and function documentation for `blas/ext/base/dsnannsumors`

### PR DESCRIPTION
**Pull request by headlessNode**
## Description

> What is the purpose of this pull request?

This pull request:

-   fix the `ndarray` function description and the parameter description
-   update logic to compute `offsetOut`
-   add the missing `float` suffx.

## Related Issues

> Does this pull request have any related issues?

No.

## Questions

> Any questions for reviewers of this pull request?

No.

## Other

> Any other information relevant to this pull request? This may include screenshots, references, and/or implementation notes.

No.

## Checklist

> Please ensure the following tasks are completed before submitting this pull request.

-   [x] Read, understood, and followed the [contributing guidelines][contributing].

* * *

@stdlib-js/reviewers

[contributing]: https://github.com/stdlib-js/stdlib/blob/develop/CONTRIBUTING.md

### CODE REVIEW

File: lib/node_modules/@stdlib/blas/ext/base/dsnannsumors/src/addon.c [hunk 1]
```diff
@@ -41,13 +43,7 @@ static napi_value addon( napi_env env, napi_callback_info info ) {
...
 
-	int io;
-	if ( strideOut < 0 ) {
-		io = -strideOut;
-	} else {
-		io = 0;
-	}
-
+	int64_t io = stdlib_strided_stride2offset( 2, strideOut );
 	double *out = Out;
```
Comment (kgryte) [inline lib/node_modules/@stdlib/blas/ext/base/dsnannsumors/src/addon.c]: @headlessNode Why do we need `out` here? Why can we not just use `Out` directly?

File: lib/node_modules/@stdlib/blas/ext/base/dsnannsumors/src/addon.c [hunk 2]
```diff
@@ -74,11 +70,10 @@ static napi_value addon_method( napi_env env, napi_callback_info info ) {
 	STDLIB_NAPI_ARGV_STRIDED_FLOAT32ARRAY( env, X, N, strideX, argv, 1 );
 	STDLIB_NAPI_ARGV_STRIDED_FLOAT64ARRAY( env, Out, 2, strideOut, argv, 4 );
 
-	int io = offsetOut;
 	double *out = Out;
```
Comment (kgryte) [inline lib/node_modules/@stdlib/blas/ext/base/dsnannsumors/src/addon.c]: @headlessNode Same question.

**Review by kgryte — COMMENTED** [review]
This PR is looking good. Just a question regarding whether a variable is actually needed.

File: lib/node_modules/@stdlib/blas/ext/base/dsnannsumors/src/addon.c (hunk 1 above)
Comment (headlessNode) [reply lib/node_modules/@stdlib/blas/ext/base/dsnannsumors/src/addon.c]: yeah `out` is redundant in this case. Thanks for pointing this out. I'll use the `Out` directly.

**Review by kgryte — APPROVED** [review]
LGTM. Thanks, @headlessNode!

**Pull request merged**